source/openapi-data-api-v1.yaml

```diff
@@ -0,0 +1,991 @@
+openapi: 3.1.0
+info:
+  version: 1.0.0
+  title: MongoDB Atlas Data API
+  description: An API to read, write, and aggregate data in MongoDB Atlas
```

Comment:
can you put 2 H2s in the description:

- 1 for configure the api with bit of explanation + link to page in docs
- 1 for authenticate requests with sentence of explanation + link to docs page

smthn like:

```md
An API to read, write, and aggregate data in MongoDB Atlas.

## Configure the Data API

Before you can use the Data API, you must configure it in Atlas App Services. Configuration includes authentication, authorization, and response types. To learn more, refer to [Set up the Data API](https://www.mongodb.com/docs/atlas/app-services/data-api/generated-endpoints/#set-up-the-data-api).

## Authenticate Requests

You must authenticate requests to the Data API based on the authentication settings in your configuration. To learn more, refer to [Authenticate Data API Requests](https://www.mongodb.com/docs/atlas/app-services/data-api/authenticate/).
```

Reply:
Added both + auth is also covered in the `security` section